src/web3_wrapper.ts

```diff
@@ -109,4 +125,10 @@ export class Web3Wrapper {
         const networkId = await promisify(this.web3.version.getNetwork)();
         return networkId;
     }
+    private async sendRawPayloadAsync(payload: Web3.JSONRPCRequestPayload): Promise<any> {
+        const sendAsync = this.web3.currentProvider.sendAsync.bind(this.web3.currentProvider);
+        const response = await promisify(sendAsync)(payload);
+        const result = response.result;
```

Comment:
What if we get a non-200 response here? Is `.result` still going to have something useful or should we return the entire response?

Reply:
If it will be non-200 there will be an exception